build.gradle

```diff
@@ -13,7 +13,11 @@ sourceCompatibility = 1.8
 
 dependencies {
     // This dependency is found on compile classpath of this component and consumers.
-    implementation 'org.checkerframework:checker:2.8.1'
+    implementation 'org.checkerframework:checker:2.8.1' // Comment this line while using local dependency of CF.
+    
+    compile files('${CHECKERFRAMEWORK}/checker/dist/checker-qual.jar')
+    compile files('${CHECKERFRAMEWORK}/checker/dist/checker.jar')
+    compile files('${CHECKERFRAMEWORK}/checker/dist/jdk8.jar')
```

Comment:
The `compile` dependency configuration is considered deprecated (https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management); you should use `implementation` instead.